decoder/body_extensions_test.go

```diff
@@ -2553,7 +2553,6 @@ resource "aws_elastic_beanstalk_environment" "example" {
 			}),
 			"",
 		},
-		// never complete dynamic as a dynamic label
```

Comment:
The reason I removed this comment here is because it doesn't seem to be adding any value, as it just duplicates the name of the test as below.